pytest_fixtures/api_fixtures.py

```diff
@@ -29,13 +29,13 @@
 
 
 @pytest.fixture(scope='session')
-def default_lce():
-    return entities.LifecycleEnvironment().search(query={'search': 'name=Library'})[0]
+def default_org():
+    return entities.Organization().search(query={'search': 'name="Default Organization"'})[0]
 
 
-@pytest.fixture(scope='module')
-def module_lce(module_org):
-    return entities.LifecycleEnvironment(organization=module_org).create()
+@pytest.fixture(scope='session')
+def default_location():
+    return entities.Location().search(query={'search': 'name="Default Location"'})[0]
 
```

Comment:
Please use the pre-defined variable for the default Location instead of doing hardcode.

```suggestion
return entities.Location().search(query={'search': f'name="{DEFAULT_LOC}"'})[0]

```